src/Command/InstallWebDirCommand.php

```diff
@@ -58,22 +71,77 @@ protected function configure()
                 'The installation root directory (defaults to the current working directory).',
                 getcwd()
             )
+            ->addOption(
+                'no-dev',
+                null,
+                InputOption::VALUE_NONE,
+                'Do not copy the app_dev.php entry point to the web folder.'
+            )
+            ->addOption(
+                'user',
+                'u',
+                InputOption::VALUE_REQUIRED,
+                'Username for the app_dev.php entry point.'
+            )
+            ->addOption(
+                'password',
+                'p',
+                InputOption::VALUE_OPTIONAL,
+                'Password for the app_dev.php entry point.'
+            )
         ;
     }
 
     /**
      * {@inheritdoc}
      */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        $user = $input->getOption('user');
+        $password = $input->getOption('password');
+
+        if (true === $input->getOption('no-dev') && (null !== $user || null !== $password)) {
+            throw new \InvalidArgumentException('Cannot set a password in no-dev mode!');
+        }
+
+        if (null === $password && null !== $user) {
+            throw new \InvalidArgumentException('Cannot set a username without password.');
+        }
+
+        if (true !== $password) {
```

Comment:
If I call `contao:install-web-dir --password=foo` the method would `return` here but the `user` option would still be missing?